src/arviz_plots/plots/psensedistplot.py

```diff
@@ -0,0 +1,282 @@
+    aes_map : mapping of {str : sequence of str}, optional
+        Mapping of artists to aesthetics that should use their mapping in `plot_collection`
+        when plotted. Valid keys are the same as for `plot_kwargs`.
+
+    plot_kwargs : mapping of {str : mapping or False}, optional
+        Valid keys are:
+
+        * One of "kde", "ecdf", "dot" or "hist", matching the `kind` argument.
+
+          * "kde" -> passed to :func:`~arviz_plots.visuals.line_xy`
+          * "ecdf" -> passed to :func:`~arviz_plots.visuals.ecdf_line`
+          * "hist" -> passed to :func: `~arviz_plots.visuals.hist`
+
+        * credible_interval -> passed to :func:`~arviz_plots.visuals.line_x`
+        * point_estimate -> passed to :func:`~arviz_plots.visuals.scatter_x`
+        * point_estimate_text -> passed to :func:`~arviz_plots.visuals.point_estimate_text`
+        * title -> passed to :func:`~arviz_plots.visuals.labelled_title`
+        * remove_axis -> not passed anywhere, can only be ``False`` to skip calling this function
+
+    stats_kwargs : mapping, optional
+        Valid keys are:
+
+        * density -> passed to kde, ecdf, ...
+        * credible_interval -> passed to eti or hdi
+        * point_estimate -> passed to mean, median or mode
+
+    pc_kwargs : mapping
+        Passed to :class:`arviz_plots.PlotCollection.wrap`
+
+    Returns
+    -------
+    PlotCollection
+
+    Examples
+    --------
+    Select a single variable and generate a point-interval plot
+
+    .. plot::
+        :context: close-figs
+
+        >>> from arviz_plots import plot_psense_dist, style
+        >>> style.use("arviz-clean")
+        >>> from arviz_base import load_arviz_data
+        >>> rugby = load_arviz_data('rugby')
+        >>> plot_psense_dist(rugby, var_names=["sd_att"], plot_kwargs={"kde":False})
+
+
+    .. minigallery:: plot_psense_dist
+
+    """
+    if sample_dims is None:
+        sample_dims = rcParams["data.sample_dims"]
+    if isinstance(sample_dims, str):
+        sample_dims = [sample_dims]
+    sample_dims = list(sample_dims)
+    if kind is None:
+        kind = rcParams["plot.density_kind"]
+    if stats_kwargs is None:
+        stats_kwargs = {}
+    else:
+        stats_kwargs = stats_kwargs.copy()
+    if plot_kwargs is None:
+        plot_kwargs = {}
+    else:
+        plot_kwargs = plot_kwargs.copy()
+    if pc_kwargs is None:
+        pc_kwargs = {}
+    else:
+        pc_kwargs = pc_kwargs.copy()
+
+    if alphas is None:
+        alphas = (0.8, 1.25)
+
+    # Here we are generating new datasets for the prior and likelihood
+    # by resampling the original dataset with the power scale weights
+    # Instead we could have weighted KDEs/ecdfs/etc
+    ds_prior = new_ds(dt, "prior", alphas, sample_dims=sample_dims)
+    ds_likelihood = new_ds(dt, "likelihood", alphas, sample_dims=sample_dims)
+    distribution = concat([ds_prior, ds_likelihood], dim="__group__").assign_coords(
+        {"__group__": ["prior", "likelihood"]}
+    )
```

Comment:
I really dislike the `__group__` after having tried it with `__sample__` in current ArviZ for `psislw`. I'd only use things like that for dimensions that are either immediately reduced right after or never make their way to the user. The `__group__` here will be a dimension on the generated plot_collection plus a dimension with aesthetics mapped to it which the user might want to modify.

Might not be the best idea, but a more compact view in a single column is also possible for example:


```python
from arviz_plots import plot_psense_dist, style

style.use("arviz-clean")

idata = load_arviz_data("rugby")

pc = plot_psense_dist(
    idata,
    var_names=["sd_def", "defs"],
    coords={"team": ["Scotland", "Wales"]},
    plot_kwargs={"kde": False},
    pc_kwargs={
        "cols": None,
        "aes": {"y": ["alpha", "__group__"], "linestyle": ["__group__"]},
        "y": [-1.2, -0.8, 0, 0, 0.8, 1.2],
    },
    aes_map={"credible_interval": ["color", "y", "linestyle"]}
)
pc.add_legend("__group__")
pc.show()
```

![imatge](https://github.com/user-attachments/assets/f37d211c-2dba-492d-b86a-1a3f4c2f9447)

The `__` needs to be added to the aesthetic definitions, and even makes it into the legend.

I can also see my choices so far being a bit too optimistic on name clashing (probably some tests had data with a `group` dimension and broke 🙈 ), but in that case I'd try to find a slightly different name not so prone to clashing with a variable or dimension. How about `psense_group`? I know it is actually longer but it doesn't give out so much of "this is a special thing" vibe

Reply:
Yeah, not happy either. I changed exactly because a clash with a test. What do you think about `group_component`, too long? And`group_c` or  `group_p`. Can we add a blank space? Too confusing?

Reply:
I like `component_group` feels a bit more natural. I generally don't care too much about things getting on the longer side, I think clear and intuitive labels are a feature worth the extra characters